rules/go_rules.build_defs

```diff
@@ -600,7 +600,7 @@ def go_get(name:str, get:str|list, repo:str='', deps:list=[], exported_deps:list
 
     # Now compile it in a separate step.
     cmd = [
-        'export GOPATH="$(find \"$TMP_DIR\" \( -name src -o -name pkg \) -exec dirname {} \; | sort | uniq | tr \'\\n\' \':\')" GO111MODULE="off"',
+        'export GOPATH="$(find \"$TMP_DIR\" \( -name src -o -name pkg \) -exec dirname {} \; | sort | uniq | tr \'\\n\' \':\' | sed \'s\/:$\/\/\')" GO111MODULE="off"',
```

Comment:
I'm not sure precisely which bit the tests are unhappy about but there's definitely something; is `sed \'s/:$//'` any happier?

Reply:
oh yeah, my mistake, changed it